x-pack/plugins/security_solution/public/detections/components/alerts_table/grouping_settings/group_stats.tsx

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiIcon } from '@elastic/eui';
+import React from 'react';
+import type {
+  BadgeMetric,
+  CustomMetric,
+} from '../../../../common/components/grouping/accordion_panel';
+import type { RawBucket } from '../../../../common/components/grouping';
+import * as i18n from '../translations';
+
+const getSingleGroupSeverity = (severity?: string) => {
+  switch (severity) {
+    case 'low':
+      return (
+        <>
+          <EuiIcon type="dot" color="#54b399" />
```

Comment:
Do we have these colors stored anywhere? Sorry to be a nit about this, I just remember the eui team updating themes before and styles being broken in places because of hard coded colors and styles. Trying to avoid that in the future if possible

Reply:
I will check with design team. Thank you for noticing.